### DISCUSSION TITLE
FaceSwap for 2 persons

### DISCUSSION THREAD

**Discussion by niiln** [lllyasviel/Fooocus#1179] [Q&A]
Is it possible to generate an image for 2 people with replacement of 2 faces. I have a photo of 2 people as an image prompt and 2 images of faces. You need to insert them when generated via face swap. Is it possible? Now you can only change 1 face. Even if you specify 2 photos as face swap, only 1 of the two will be generated and put into 1 person. Perhaps I'm doing something wrong. Please help.

**Comment by Graf63** [comment]
Good question, but so far, personally, I cut the image in 2 and I do the face swaps.

**Comment by hqbasslab** [comment]
2 faces at once probably will be mixed and make 1 face.. but for me don't work even 2 images as faceswap. It's loading controlnet models and freeze everything forcing to reset whole colab environment:(
